src/commands/ExecCommand.js

```diff
@@ -18,17 +18,19 @@ export const builder = {
     group: "Command Options:",
     describe: "Bail on exec execution when the command fails within a package",
     type: "boolean",
-    default: true,
+    default: undefined,
```

Comment:
Yes, I think this would be a great example to reference when raising this upstream, which I also think would be a good idea.